src/pages/docs/degenhack/mainnet-version.md

```diff
@@ -271,6 +271,7 @@ The JS SDK consists of the following components:
 
 **Gvmkit-build** - a tool designed to create custom images for virtual machines on provider nodes within the Golem Network. It specifically converts Dockerfiles into Golem-compatible VM images, known as GVMI.
 
-You find a list of available instructions for Golem tools [here](https://docs.golem.network/docs/creators/javascript/examples/tools).
+You can find a list of available instructions for Golem tools: [yagna](https://docs.golem.network/docs/creators/tools/yagna)
+and [gvmkit-build](https://docs.golem.network/docs/creators/tools/gvmkit).
```

Comment:
I'd move these links so that they are part of their respective paragraphs above, like this:

```
**Yagna** - the foundational service that allows users to connect with and interact with the Golem Network. You're welcome to [learn more about its installation and usage](https://docs.golem.network/docs/creators/tools/yagna).

**Gvmkit-build** - a tool designed to create custom images for virtual machines on provider nodes within the Golem Network. It specifically converts Dockerfiles into Golem-compatible VM images, known as GVMI. For usage, have a look at: [gvmkit-build](https://docs.golem.network/docs/creators/tools/gvmkit).
```

Reply:
will not update, there is no separate section for gvmkit-built, so the tool section must stay, the whole article is for one time event, (already passed),